### DISCUSSION TITLE
Table components

### DISCUSSION THREAD

**Discussion by Andre1275** [Budibase/budibase#13619] [Help] [answered]
Hello 

Haven't done anything for a few days now. 
After an update, the simple table control has disappeared. 
Does it no longer exist?

Greetings Andre

**Comment by ConorWebb96** [comment, marked as answer]
https://github.com/Budibase/budibase/discussions/13428

**Comment by Andre1275** [comment]
Hello Conor 

I have read the various threads.
I'm a bit sad that the old component has been replaced.
In one thread the question was asked about changing the color to a column, unfortunately this was not answered. Is there a solution?

Greetings Andre
